events/views.py

```diff
@@ -27,7 +23,7 @@ def saveEvent(request):
         capacity = request.POST['capacity']
     except (KeyError, Event.DoesNotExist):
         # Redisplay the question voting form.
-        return redirect('events:my-events')
```

Comment:
I wonder why the entire "my-event" page is deleted and save/create/delete events all direct to "all-events" page? I understand that for now we don't have log-in, so there's not really a difference between "all events" and "my events". But just as a placeholder, wouldn't delete normally happen on the "my-event" page and create/update also direct to the "my event" list? However, if this is for technical reasons, I understand.

Reply:
Discussion we had was -
1. Delete and update button will only be shown on events created by me in events page
2. in events page we will have a filter "created by me"

Reason being maintaining 2 pages is technically and product wise more complex

Reply:
Cool that makes sense. I'm still wondering though, we may need the "my event" page to be a separate page to show all the events that are created/also added by the user (a bit like a user's own memo list). I'm imagining it has a different UI then the "all events" page. We can discuss that more in next sprint.